src/module/reservation.js

```diff
@@ -0,0 +1,142 @@
+const showReservations = (e) => {
+  const popup = document.createElement('div');
+  document.body.appendChild(popup);
+  popup.classList.add('popUp', 'popUpContainer');
+  popup.style.height = '100%';
+  popup.style.width = '100%';
+  popup.style.position = 'fixed';
+  popup.style.top = '0';
+  popup.style.left = '0';
+  popup.style.backgroundColor = '#d3d3d3';
+  popup.style.zIndex = '1';
+  popup.style.display = 'grid';
+  popup.style.borderRadius = '10px';
+  popup.style.boxShadow = '0px 0px 10px rgba(0,0,0,0.5)';
+  popup.style.overflowY = 'auto';
+
+  const idMeal = e.target.parentElement.parentElement.children[0].children[0].innerHTML;
+  const url = `https://www.themealdb.com/api/json/v1/1/lookup.php?i=${idMeal}`;
+  const getData = async () => {
+    const response = await fetch(url);
+    const data = await response.json();
+    return data;
+  };
+  const lookup = getData();
+  lookup.then((data) => {
+    const mealName = data.meals[0].strMeal;
+    const mealImage = data.meals[0].strMealThumb;
+    const tags = data.meals[0].strTags;
+    popup.classList.add('popup');
+    popup.innerHTML = `
+    <img class = "popup-Images" src="${mealImage}" alt="Food Image">
+    <div class="popup-inner" id="start">
+        <article class="popup-header">
+            <button id="end"><svg class ="endALl" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512"><!--! Font Awesome Pro 6.2.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2022 Fonticons, Inc. --><path d="M310.6 150.6c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0L160 210.7 54.6 105.4c-12.5-12.5-32.8-12.5-45.3 0s-12.5 32.8 0 45.3L114.7 256 9.4 361.4c-12.5 12.5-12.5 32.8 0 45.3s32.8 12.5 45.3 0L160 301.3 265.4 406.6c12.5 12.5 32.8 12.5 45.3 0s12.5-32.8 0-45.3L205.3 256 310.6 150.6z"/></svg></button>
```

Comment:
We could add the link for Font Awesome on the header of our HTML template to avoid bloating tag elements with web links.

Reply:
Hello @PraisesPJMT, I've found from multiple web sources that using font awesome links tends to be slow as compared to using plain SVGs, SVGs act like images and vectors.